packages/teleterm/src/ui/StatusBar/ShareFeedback/ShareFeedbackForm.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import FieldInput from 'shared/components/FieldInput';
+import { requiredField } from 'shared/components/Validation/rules';
+import { FieldTextArea } from 'shared/components/FieldTextArea';
+import { Text } from 'design';
+import Toggle from 'teleport/components/Toggle';
+import { ShareFeedbackFormValues } from './types';
+
+interface ShareFeedbackFormProps {
+  formValues: ShareFeedbackFormValues;
+
+  setFormValues(values: ShareFeedbackFormValues): void;
+}
+
+export function ShareFeedbackForm({
+  formValues,
+  setFormValues,
+}: ShareFeedbackFormProps) {
+  function updateFormField<T extends keyof ShareFeedbackFormValues>(
+    field: T,
+    value: ShareFeedbackFormValues[T]
+  ) {
+    setFormValues({ ...formValues, [field]: value });
+  }
+
+  return (
+    <>
+      <FieldInput
+        mt={3}
+        type="email"
+        label="Email Address"
+        autoFocus
+        css={`
+          input {
+            font-size: 14px;
+          }
+        `}
+        rule={requiredField('Email is required')}
+        value={formValues.email}
+        onChange={e => updateFormField('email', e.target.value)}
+      />
+      <FieldTextArea
+        mt={1}
+        label="Any suggestions?"
+        textAreaCss={`
+                font-size: 14px;
+              `}
+        value={formValues.feedback}
+        onChange={e => updateFormField('feedback', e.target.value)}
+        placeholder="Type your suggestions here"
+      />
+      <Toggle
+        isToggled={formValues.newsletterEnabled}
+        onToggle={() => {
+          updateFormField('newsletterEnabled', !formValues.newsletterEnabled);
+        }}
+      >
+        <Text ml={2} color="text.primary">
+          Sign me up for the newsletter
+        </Text>
+      </Toggle>
+      <Toggle
+        isToggled={formValues.salesContactEnabled}
+        onToggle={() => {
+          updateFormField(
+            'salesContactEnabled',
+            !formValues.salesContactEnabled
+          );
+        }}
+      >
+        <Text ml={2} color="text.primary">
+          I want to be contacted by the sales team
```

Comment:
Sasha posted this copy on #dev-connect:

> I want your sales team to demo me Teleport Enterprise features

It makes the form quite wide, perhaps we should put some max-width on it? The label would need to be fixed though because the spacing is broken when the label text is split into two lines.

Reply:
Done.
<img width="394" alt="image" src="https://user-images.githubusercontent.com/20583051/173552228-bd77c5ae-4817-40ce-8eff-524f88f25f12.png">

Reply:
Cool, did you do anything special to make those two checkbox labels line up or did I just get used to them looking like that? I couldn't find anything in the code. 🤔

Reply:
Actually……………………… could you put nbsp between Teleport and Enterprise?

Reply:
tbh I cannot decide which version looks worse; I'd say the one without nbsp looks better? At least it's not cut in an awkward way.

![feedback form with nbsp](https://user-images.githubusercontent.com/27113/173567052-94b482bc-f776-4340-84d8-46398a5a4f86.png)

Reply:
I'd vote for the one with `&nbsp;` - if we can avoid breaking the product name, let's do it.

>did you do anything special to make those two checkbox labels line up?
Hmm the only thing which I did was decreasing the `line-height` for the second label.

Reply:
> I'd vote for the one with `&nbsp;` - if we can avoid breaking the product name, let's do it.

👍